esp32/modules/installer.py

```diff
@@ -45,6 +45,18 @@ def show_description(active):
 		text.text(packages[options.selected_index()]["description"])
 		ugfx.flush()
 
+def move_sel(active,dir_):
+	if active:
+		sel = options.selected_index()
+		print("Ingoing:", sel)
```

Comment:
(and several other print statements)
This looks like debugging. Do we really need it?